composables/useScenarioOptions.ts

```diff
@@ -1,27 +1,61 @@
 import { type Parameter, type ParameterOption, TypeOfParameter } from "~/types/parameterTypes";
-import { paramOptsToSelectOpts } from "~/components/utils/parameters";
+import { getRangeForDependentParam, paramOptsToSelectOpts } from "~/components/utils/parameters";
+import { humanReadableInteger } from "~/components/utils/formatters";
 
 export default (parameterAxis: MaybeRefOrGetter<Parameter | undefined>) => {
   const appStore = useAppStore();
 
+  const axis = computed(() => toValue(parameterAxis));
+
   const baselineOption = computed(() => {
-    const axis = toValue(parameterAxis);
-    if (!appStore.currentScenario.parameters || !axis) {
+    if (!appStore.currentScenario.parameters || !axis.value) {
       return undefined;
-    } else if (axis.parameterType === TypeOfParameter.Numeric) {
-      const baselineValue = appStore.currentScenario.parameters[axis.id];
+    } else if (axis.value.parameterType === TypeOfParameter.Numeric) {
+      const baselineValue = appStore.currentScenario.parameters[axis.value.id];
       // TODO: (jidea-229) description should say whether the value is a default, min, max; or empty if user-provided.
```

Comment:
Are you planning on implementing this in this ticket? Could spin it out into another.

Reply:
I accidentally left this in. It shouldn't be done, since this is the baseline option, not one that a user can actually select, so they'll never see it in a menu.